database/redis/README.md

```diff
@@ -0,0 +1,95 @@
+## Redis
+
+Redis is an open source (BSD licensed), in-memory data structure store, used as a database, cache and message broker. It supports data structures such as strings, hashes, lists, sets, sorted sets with range queries, bitmaps, hyperloglogs, geospatial indexes with radius queries and streams.
+
+In this example, we will generate a Singularity container running Redis server v5.0.0 over Debian 9 (Stretch).
+
+To run this example you will need:
+
+	- Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+	- A text editor, like: `micro`, `vim`, or `nano`.
+	- Root access.
```

Comment:
I would replace the whitespace (lines 9, 10, 11) with two spaces, so its not in a code block.